src/main/features/core/settings/index.ts

```diff
@@ -48,3 +48,8 @@ ipcMain.handle('password-set', (_event, password: string, server: string) => {
     }
     return false;
 });
+
+ipcMain.on('theme-set', (_event, theme: 'dark' | 'light' | 'system') => {
```

Comment:
Would be nice to have a shared type for the theme `'dark' | 'light' | 'system'` since it's now being used in 3 places.